From 53dea9d87a6d0b26a016970500e60d8902e5e738 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Tue, 2 Jul 2013 16:28:38 -0400
Subject: [PATCH] wr: allow ifconfig to read/search /var/run

The following logs show when libvirtd starts, and transitions to
ifconfig_t:
type=1400 audit(1370230373.687:10): avc:  denied { read } for  pid=2053 comm="ip" name="run" dev="sda6" ino=263292 scontext=unconfined_u:system_r:ifconfig_t:s0-s0:c0.c1023 tcontext=system_u:object_r:var_run_t:s0 tclass=lnk_file
type=1400 audit(1370230373.707:11): avc:  denied  { search } for pid=2053 comm="ip" name="run" dev="tmpfs" ino=13591 scontext=unconfined_u:system_r:ifconfig_t:s0-s0:c0.c1023 tcontext=system_u:object_r:var_run_t:s0 tclass=dir

Upstream-Status: pending

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 policy/modules/system/sysnetwork.te |    1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/sysnetwork.te b/policy/modules/system/sysnetwork.te
index ed363e1..8a20cad 100644
--- a/policy/modules/system/sysnetwork.te
+++ b/policy/modules/system/sysnetwork.te
@@ -281,6 +281,7 @@ domain_use_interactive_fds(ifconfig_t)
 
 files_read_etc_files(ifconfig_t)
 files_read_etc_runtime_files(ifconfig_t)
+files_list_pids(ifconfig_t)
 
 fs_getattr_xattr_fs(ifconfig_t)
 fs_search_auto_mountpoints(ifconfig_t)
-- 
1.7.9.7

